### PR TITLE
audero-sticky

### PR DESCRIPTION
**Pull request by AurelioDeRosa**

### CODE REVIEW

File: files/audero-sticky/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Aurelio De Rosa"
+github = "https://github.com/AurelioDeRosa/audero-sticky"
+homepage = "https://github.com/AurelioDeRosa/audero-sticky"
+description = "Audero Sticky is a polyfill for the CSS position: sticky with no dependencies and support for multiple module systems."
+mainfile = "dist/audero-sticky.min.js"
```
Comment (jimaek) [inline files/audero-sticky/info.ini]: Just `audero-sticky.min.js` here

**Comment by AurelioDeRosa** [comment]
Done, thank you!

**Comment by jimaek** [comment]
@jsdelirbot squash and merge

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**